source/includes/fact-verifier-unsupported.rst

```diff
@@ -0,0 +1,16 @@
+The verifier doesn't check the following namespaces: 
+
+- Capped collections
+- Collections with TTL indexes
+- Collections that don't use the default collation
+- Views
+
+The verifier doesn't check the following collection features:
+
+- Collection metadata
```

Comment:
It also does not verify sharding configurations

Reply:
Fixed